Assets/Scripts/Controllers/CharacterController-DESKTOP-7IQNCQ4.cs

```diff
@@ -7,13 +7,16 @@ public class CharacterController : MonoBehaviour
 {
     private Rigidbody2D _rb;
     private Vector2 _movement;
+    private CapsuleCollider2D _cc;
 
+    [SerializeField] private LayerMask groundMask;
```

Comment:
`groundMask` -> `_groundMask`.

non-public class variable members should be prefixed with "_"